File: Programmers/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
import java.util.*;

class Solution {
    public String solution(String s) {
        StringBuilder answer = new StringBuilder();
        boolean isStart = true;

        for (char c : s.toCharArray()) {
            if (isStart) {
                answer.append(Character.isLetter(c) ? Character.toUpperCase(c) : c);
            } else {
                answer.append(Character.isLetter(c) ? Character.toLowerCase(c) : c);
            }
            
            isStart = (c == ' ');
        }

        return answer.toString();
    }
}

File: Programmers/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    bool isStart = true;
    
    for (int i = 0; i < s.size(); i++)
    {
        char &c = s[i];
        
        if (isStart)
        {
            if (isalpha(c))
                c = toupper(c);
        } 
        else
        {
            if (isalpha(c))
                c = tolower(c);
        }
        
        isStart = (c == ' ');
    }
    
    return s;
}